### DISCUSSION TITLE
os 22631.3527 (windows 11 23H2) 点击开始菜单中的头像无法弹出注销和用户对话框

### DISCUSSION THREAD

**Discussion by zbrahimovic** [valinet/ExplorerPatcher#3190] [General]
os 22631.3527 (windows 11 23H2) 点击开始菜单中的用户头像无法弹出注销和用户对话框，用的是22621.3527.65.1，

**Comment by zbrahimovic** [comment]
![屏幕截图 2024-04-27 183119](https://github.com/valinet/ExplorerPatcher/assets/153735805/7cb55617-863c-4dc8-9083-fc5414aab4ed)
this  position  click then can't see user and log off

**Comment by pyrates999** [comment]
Try to open up task manager and kill explorer.exe and then run task manager again and run explorer.exe

Does that fix it?

Please include what settings you changed in explorer patcher properties application.  These should be screen shots.  If you can, please change it to english first.

Please include any other software you have installed that modifies the UI of windows.

**Comment by pyrates999** [comment]
Just found it was fixed here: https://github.com/valinet/ExplorerPatcher/issues/2977#event-12632038926

Please install the latest pre-release here: https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3527.65.1_d11445a

**Reply by zbrahimovic** [reply]
thank you for your remind ，I will try it
